tensorflow/stream_executor/cuda/cuda_blas.cc

```diff
@@ -299,6 +305,24 @@ static string ToString(cublasStatus_t status) {
   }
 }
 
+// Decide whether to enable TENSOR_OP_MATH
+static bool TensorOpMathEnabled() {
+  static bool is_enabled = [] {
+    bool is_disabled;
+    if (getenv("TF_DISABLE_CUBLAS_TENSOR_OP_MATH") != nullptr) {
+      TF_CHECK_OK(
+          tensorflow::ReadBoolFromEnvVar("TF_DISABLE_CUBLAS_TENSOR_OP_MATH",
+                                         /*default=*/false, &is_disabled));
+    } else {
+      TF_CHECK_OK(tensorflow::ReadBoolFromEnvVar("TF_DISABLE_TENSOR_OP_MATH",
```

Comment:
In general, I'm not a fan of hierarchical flags, since their relationship is not clearly documented. If the name is confusing, why not just change the other one into "TF_DISABLE_CUDNN_TENSOR_OP_MATH"?

Reply:
Removed TF_DISABLE_TENSOR_OP_MATH.